library/ecp.c

```diff
@@ -2059,6 +2064,58 @@ int mbedtls_ecp_check_pub_priv( const mbedtls_ecp_keypair *pub, const mbedtls_ec
     return( ret );
 }
 
+#if defined(MBEDTLS_ASN1_WRITE_C) && defined(MBEDTLS_OID_C)
+int mbedtls_ecp_ansi_write_group( const mbedtls_ecp_group *grp,
```

Comment:
Like before, I would prefer to use ASN.1 in the name of the function instead of ANSI.